app/app_test.go

```diff
@@ -6,45 +6,40 @@ import (
 	"testing"
 
```

Comment:
I see a `WARNING: DATA RACE` in the tests. Does anyone have an idea how to fix this?

Reply:
Oh well, Robert's grammar change (https://github.com/regen-network/regen-ledger/pull/219#discussion_r557418453) made the error go away 👍